### PR TITLE
merged GetArticle into GetText and fixes in lamdba_function.py to call the python functions

### PR DESCRIPTION
**Pull request by ANA-POTJE**

### CODE REVIEW

File: src/getText.py [hunk 1]
```diff
@@ -1,32 +1,56 @@
...
         article = Article(url)
         article.download()
         article.parse()
     ### Exception - e.g if URL is "valid" but inexistent, no text will be retrieved
     except Exception as e: 
         #print(e)
-        return('-1')
-    
+        return  {'text': '-1',
+                 'header': '',
```
Comment (jsjohnstone) [inline src/getText.py:38]: Wonder if we should standardise these to make it easier on the frontend, and instead return an error object like:
`{"error" : "The article could not be downloaded." }`

Comment (jsjohnstone) [reply src/getText.py:38]: Maybe ignore this and it could be a future fix.

File: src/lambda_function.py [hunk 2]
```diff
@@ -21,17 +22,46 @@ def lambda_handler(event, context):
...
     
-    credibilityresult = cr.getCredibilityScore(url)
-    
+    #### Define the object skeleton
     object = {
         "url" : url,
-        "results" : [
-            credibilityresult
-        ]
+        "article" : {},
```
Comment (jsjohnstone) [inline src/lambda_function.py:28]: I think we should actually remove this variable initialisation - it's created as part of the good or error case down below.

**Review by jsjohnstone — CHANGES_REQUESTED** [review]
Just one change around initialising the 'article' object :)

File: src/lambda_function.py [hunk 3]
```diff
@@ -21,17 +22,46 @@ def lambda_handler(event, context):
...
+    try:
+        sentanalysisresult = sa.sentimentAnalysis(url)
+        object['article'] = {'header': sentanalysisresult['header'], 
+                             'summary': sentanalysisresult['summary'],
+                             'keywords': sentanalysisresult['keywords']}
+        object['results'].append({'type': 'polarity',     'outcome': sentanalysisresult['polarity']})
+        object['results'].append({'type': 'subjectivity', 'outcome': sentanalysisresult['subjectivity']})
+    except Exception as e:
+        object['article'] = {'error': 'The article could not be retrieved.'}
+        object['results'].append({'type': 'polarity',     'error': 'no data available'})
```
Comment (jsjohnstone) [inline src/lambda_function.py:47]: We should standardise how objects/results are returned - it's slightly different between components.

**Review by jsjohnstone — CHANGES_REQUESTED** [review]

**Comment by ANA-POTJE** [comment]
Closing this PR, as changes requested will be available in the next version of this branch.

**Pull request closed without merging**